django/utils/log.py

```diff
@@ -56,6 +56,11 @@
             'level': 'ERROR',
             'propagate': False,
         },
+        'django.template': {
+            'handlers': ['console'],
+            'level': 'WARNING',
```

Comment:
I'd log a DEBUG message if DEBUG is True instead of WARNING. You can use the RequireDebugTrue filter.